src/variable.rs

```diff
@@ -262,12 +285,16 @@ pub fn variable() -> VariableDefinition {
 #[derive(Default)]
 pub struct ProblemVariables {
     variables: Vec<VariableDefinition>,
+    initial_count: usize,
 }
 
 impl ProblemVariables {
     /// Create an empty list of variables
     pub fn new() -> Self {
-        ProblemVariables { variables: vec![] }
+        ProblemVariables {
+            variables: vec![],
+            initial_count: 0,
```

Comment:
Is this adding to the size? If yes, should we just change the test now?

Reply:
I removed the test. It covers implementation details rather than correctness and IMO it makes little sense to check for this.